infrastructure/terraform/main/files/values.yaml

```diff
@@ -0,0 +1,5 @@
+ingress-controller:
+  host: ${host}.${hosted_zone}
+security:
+  allowedOrigins: "*"
+apiHost: https://${host}.${hosted_zone}
```

Comment:
Shouldn't we name this as `airy.yaml` so that it is inline with the current usage?

Perhaps instead of having it under files, we can generate is as a result of the completion of the `core` module (if the file doesn't exist in the workspace).

Reply:
done!